### PR TITLE
Move timeout height into governance

### PR DESCRIPTION
**Pull request by antstalepresh**

### CODE REVIEW

File: x/bech32ibc/client/cli/tx.go [hunk 1]
```diff
@@ -64,7 +71,18 @@ func NewCmdSubmitUpdateHrpIbcRecordProposal() *cobra.Command {
...
+			heightOffset, err := cmd.Flags().GetUint64(IcsToHeightOffset)
+			if err != nil {
+				return err
+			}
+
+			durationOffsetText, err := cmd.Flags().GetString(IcsToTimeoutOffset)
+			if err != nil {
+				return err
+			}
+			durationOffset, err := time.ParseDuration(durationOffsetText)
```
Comment (ValarDragon) [inline x/bech32ibc/client/cli/tx.go:83]: Lets require that one of these be set, e.g. either duration non-zero or height offset non-zero?

File: x/bech32ibc/client/cli/tx.go [hunk 2]
```diff
@@ -82,6 +100,8 @@ func NewCmdSubmitUpdateHrpIbcRecordProposal() *cobra.Command {
 	cmd.Flags().String(cli.FlagTitle, "", "title of proposal")
 	cmd.Flags().String(cli.FlagDescription, "", "description of proposal")
 	cmd.Flags().String(cli.FlagDeposit, "", "deposit of proposal")
+	cmd.Flags().Uint64(IcsToHeightOffset, 0, "the offset of height to expire on target chain")
```
Comment (ValarDragon) [inline x/bech32ibc/client/cli/tx.go]:
```suggestion
	cmd.Flags().Uint64(IcsToHeightOffset, 0, "timeout for IBC routed packets through this channel, in blocks. A value of X here, means that if a packet is attempted to get relayed at counter-party chain height of N, and fails to be ack'd by height N+X, the packet will bounce back to the source chain.")
```

File: x/bech32ibc/client/cli/tx.go (hunk 1 above)
Comment (ValarDragon) [reply x/bech32ibc/client/cli/tx.go:83]: We should also require this in the msg's validate basic

**Review by ValarDragon — APPROVED** [review]

File: x/bech32ibc/client/cli/tx.go (hunk 1 above)
Comment (antstalepresh) [reply x/bech32ibc/client/cli/tx.go:83]: Done.

**Pull request merged**